docs/src/content/docs/it/guides/authoring-content.md

```diff
@@ -89,6 +105,8 @@ Posso collegarmi alla [mia conclusione](#conclusione) che si trova più in basso
 
 Titoli di livello 2 (`<h2>`) e di livello 3 (`<h3>`) verranno inclusi automaticamente nella tabella dei contenuti.
 
+Scopri come Astro processa gli `id` di heading nella [documentazione di Astro](https://docs.astro.build/it/guides/markdown-content/#heading-ids)
+
 ## Aside
```

Comment:
TBF initially I was going to leave it as Aside, because I couldn't find any translation that made sense. But I found a translation that conveys well enought the concept of "Aside": Avvisi. In english the direct translation is "alerts" but in this context I think "Avvisi" is the best next thing.